pandas/core/arrays/categorical.py

```diff
@@ -2506,16 +2505,90 @@ class CategoricalAccessor(PandasDelegate, PandasObject, NoNewAttributesMixin):
 
     Examples
     --------
+    >>> s = pd.Series(list("aabc")).astype("category")
+    >>> s
+    0    a
+    1    a
+    2    b
+    3    c
+    dtype: category
+    Categories (3, object): [a, b, c]
+
     >>> s.cat.categories
-    >>> s.cat.categories = list('abc')
-    >>> s.cat.rename_categories(list('cab'))
-    >>> s.cat.reorder_categories(list('cab'))
-    >>> s.cat.add_categories(['d','e'])
+    Index(['a', 'b', 'c'], dtype='object')
+
+    >>> s.cat.categories = list("bcd")
```

Comment:
I personally think we should remove this example ... (seems bad practice to me to modify the categorical like this in place)